servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -3892,56 +3914,17 @@ void executePutDomainMeta(ResourceContext ctx, Domain domain, DomainMeta meta,
     private boolean processDomainTags(ObjectStoreConnection con, Map<String, TagValueList> domainTags,
             Domain originalDomain, final String domainName, boolean updateDomainLastModTimestamp) {
 
-        if (originalDomain == null || originalDomain.getTags() == null || originalDomain.getTags().isEmpty()) {
-            if (domainTags == null || domainTags.isEmpty()) {
-                // no tags to process
-                return true;
-            }
-            if (!con.insertDomainTags(domainName, domainTags)) {
-                return false;
-            }
+        BiFunction<ObjectStoreConnection, Map<String, TagValueList>, Boolean> insertOp = (ObjectStoreConnection c, Map<String, TagValueList> tags) -> c.insertDomainTags(domainName, tags);
```

Comment:
wasn't this already merged?

Reply:
it was only for role and group i added this changed for domain as well